tests/Unit/Domain/CoordinateMaps/Test_CoordinateMap.cpp

```diff
@@ -888,6 +893,87 @@ void test_coordinate_maps_are_identity() {
+      {"trans", f_of_t}};
+  const CoordMapsTimeDependent::Translation trans_map{};
+
+  // affine(x) = 1.5 * x + 5.5
+  domain::CoordinateMaps::Affine affine_map{-1., 1., 4., 7.};
+
+  const auto time_dependent_map_first =
+      make_coordinate_map<Frame::Logical, Frame::Inertial>(trans_map,
+                                                           affine_map);
+  const auto time_dependent_map_second =
+      make_coordinate_map<Frame::Logical, Frame::Inertial>(affine_map,
+                                                           trans_map);
+
+  const tnsr::I<double, 1, Frame::Logical> tnsr_double_logical{{{3.2}}};
+  const tnsr::I<DataVector, 1, Frame::Logical> tnsr_datavector_logical{
+      DataVector{{-4.3, 10.1, -3.5}}};
+
+  const tnsr::I<double, 1, Frame::Inertial> tnsr_double_inertial_1{{{39.34}}};
+  const tnsr::I<double, 1, Frame::Inertial> tnsr_double_inertial_2{{{29.66}}};
+
+  const tnsr::I<DataVector, 1, Frame::Inertial> tnsr_datavector_inertial_1{
+      DataVector{28.09, 49.69, 29.29}};
+  const tnsr::I<DataVector, 1, Frame::Inertial> tnsr_datavector_inertial_2{
+      DataVector{18.41, 40.01, 19.61}};
+
+  CHECK_ITERABLE_APPROX(
+      time_dependent_map_first(tnsr_double_logical, final_time, f_of_t_list),
+      tnsr_double_inertial_1);
+
+  CHECK_ITERABLE_APPROX(
+      time_dependent_map_second(tnsr_double_logical, final_time, f_of_t_list),
+      tnsr_double_inertial_2);
+
+  CHECK_ITERABLE_APPROX(time_dependent_map_first(tnsr_datavector_logical,
+                                                 final_time, f_of_t_list),
+                        tnsr_datavector_inertial_1);
+  CHECK_ITERABLE_APPROX(time_dependent_map_second(tnsr_datavector_logical,
+                                                  final_time, f_of_t_list),
+                        tnsr_datavector_inertial_2);
+
+  CHECK_ITERABLE_APPROX(*(time_dependent_map_first.inverse(
+                            tnsr_double_inertial_1, final_time, f_of_t_list)),
+                        tnsr_double_logical);
+  CHECK_ITERABLE_APPROX(*(time_dependent_map_second.inverse(
+                            tnsr_double_inertial_2, final_time, f_of_t_list)),
+                        tnsr_double_logical);
+
+  CHECK(time_dependent_map_first
+            .jacobian(tnsr_double_logical, final_time, f_of_t_list)
+            .get(0, 0) == 1.5);
+  CHECK(time_dependent_map_first
+            .inv_jacobian(tnsr_double_logical, final_time, f_of_t_list)
+            .get(0, 0) == 2. / 3.);
+  CHECK_ITERABLE_APPROX(
+      time_dependent_map_first
+          .jacobian(tnsr_datavector_logical, final_time, f_of_t_list)
+          .get(0, 0),
+      (DataVector{1.5, 1.5, 1.5}));
+  CHECK_ITERABLE_APPROX(
+      time_dependent_map_first
+          .inv_jacobian(tnsr_datavector_logical, final_time, f_of_t_list)
+          .get(0, 0),
+      (DataVector{2. / 3., 2. / 3., 2. / 3.}));
+
+  test_serialization(time_dependent_map_first);
```

Comment:
Can you also do all the above `CHECK`s with the deserialized map to make sure that nothing breaks in a subtle manner?